crates/services/compression/src/storage.rs

```diff
@@ -46,5 +47,8 @@ pub type RegistryIndex = Merkleized<registry_index::RegistryIndex>;
 /// Merkleized Timestamps table type alias
 pub type Timestamps = Merkleized<timestamps::Timestamps>;
 
-/// Re-export to match api
-pub use compressed_blocks::CompressedBlocks;
+/// Merkleized compressed blocks table type alias
+pub type CompressedBlocks = Merkleized<compressed_blocks::CompressedBlocks>;
```

Comment:
Using `Merklized` means that we're using the SMT blueprint. However, do we need SMTs for this table? The normal block roots are using BMTs instead, which I'd assume is sufficient in this case as well. Or am I forgetting something?

Reply:
resolved on slack